openmls/src/messages/tests/test_welcome.rs

```diff
@@ -162,26 +163,24 @@ fn test_welcome_ciphersuite_mismatch(
     let group_info_bytes = welcome_key
         .aead_open(backend, welcome.encrypted_group_info(), &[], &welcome_nonce)
         .expect("Could not decrypt GroupInfo.");
-    let mut group_info = GroupInfo::tls_deserialize(&mut group_info_bytes.as_slice())
-        .expect("Could not deserialize GroupInfo.");
```

Comment:
Expects like this can easily get out-of-sync and the `expect` doesn't really add information here. Thus, I used `unwrap` instead.